### PR TITLE
move working dirs to .wercker

### PR DESCRIPTION
**Pull request by termie**

### CODE REVIEW

**Comment by termie** [comment]
addresses #8

**Comment by termie** [comment]
actually i have a couple things i want to do to this before merge

**Comment by termie** [comment]
lookin ready now

File: cmd/runner.go [hunk 1]
```diff
@@ -196,6 +205,11 @@ func (p *Runner) EnsureCode() (string, error) {
 				if util.ContainsString(ignoreFiles, abspath) {
 					ignores = append(ignores, file.Name())
 				}
+
+				// TODO(termie): remove this warning after a while
+				if util.ContainsString(deprecatedPaths, abspath) {
```
Comment (faiq) [inline cmd/runner.go]: I'm under the opinion that you should keep all the old ignore files in ignoreFiles, and still have this warning.

**Comment by hatchan** [comment]
:lgtm:

File: cmd/runner.go (hunk 1 above)
Comment (termie) [inline cmd/runner.go]: if they old ignore files are in there, the user has no way to deal with having their own _builds directory, previously they could change working-dir to not conflict but with this change that would no longer work

**Pull request merged**